api/filesys/fsnode.go

```diff
@@ -37,9 +38,9 @@ type fsNode struct {
 	// if this node is a file, this is the content.
 	content []byte
 
-	// if this node is a file, this tracks whether or
-	// not it is "open".
-	open bool
+	// if offset is nil the file is open and buf tracks
```

Comment:
This says "nil means open" but the changes below seem to contradict this by replacing `open = false` with `offset = nil`.

Reply:
True! Forgot a "not" before nil. Fixed